core/domain/skill_services.py

```diff
@@ -486,7 +486,10 @@ def apply_change_list(skill_id, change_list, committer_id):
                     skill.update_explanation(change.new_value)
                 elif (change.property_name ==
                       skill_domain.SKILL_CONTENTS_PROPERTY_WORKED_EXAMPLES):
-                    skill.update_worked_examples(change.new_value)
+                    worked_examples_list = [
+                        state_domain.SubtitledHtml.from_dict(worked_example)
```

Comment:
I think this is where the test failures are coming from. The error from Travis-CI is:
```
NameError: global name 'state_domain' is not defined
```
To me, this looks like `state_domain` might need to be imported

Reply:
Done!